docs/site/pages/assets/dsl.mdx

```diff
@@ -0,0 +1,126 @@
+---
+title: Writing DSL Components
+---
+
+# Creating TSX Components
+
+In order to take advantage of the auto-completion and validation of TypeScript types, asset libraries can export a component library for content authoring. Creating components isn't much different than writing a React component for the web. The primative elements uses the [react-json-reconciler](https://github.com/intuit/react-json-reconciler) to create the JSON content tree, with utilities to make it quick and painless to create new asset-components.
+
+
+## Creating a Basic Component
+
+The `Asset` component from the `@player-tools/dsl` package is the quickest way to create a new component. The `Asset` component will take all the Asset's properties and convert them to their equivalent JSON representation when serialized.
```

Comment:
Good call. It is a bit confusing, almost intentionally, since we want `Asset` to always mean the base that we provide. In the example below we pull in `Asset` from player types for the base TS definition of the Asset's type *and* we pull in `Asset` from the DSL package for the base of the Asset's DSL component. I can add a note explaining that.